spec/models/flavours_spec.rb

```diff
@@ -0,0 +1,47 @@
+describe Flavours do
+  subject(:flavours_instance) { described_class.new(cache: cache_double, db_collection: db_collection) }
+  let(:db_collection) { 'appdb-test' }
+  let(:cache_double) { instance_double('Utils::MongodbCache') }
+  let(:cache_data) do
+    file = File.read('spec/stub_data/appdb_stub.json')
+    JSON.parse(file)
+  end
+  let(:appliance_id) { '511a2916-d480-4b56-b788-c76cb8c57509' }
+  let(:site_id) { '247G0:4450G0' }
+  let(:flavour_id) { 'cmVzb3VyY2VfdHBsIzE=' }
+  before { allow(cache_double).to receive(:cache_fetch).with(db_collection) }
+
+  describe '#list' do
+    context 'db contains data' do
```

Comment:
Error is equivalent with ('missing', 'missing') version, since appliance is always tested first. However the test can be added. Should i allso add all combinations for #show?